Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const makeDurationHours = tasks.map(elem => elem.duration / 60);
+  const TwoHoursOrBigger = makeDurationHours.filter(elem => elem >= 2);
+  let sum = 0;
```

Comment:
I see you played it safe and didn't go for the bonus round of summing everything using `reduce`. 😉 

I would definitely recommend though to try to use reduce here instead of this big for loop.